django/contrib/admindocs/views.py

```diff
@@ -60,11 +55,11 @@ class TemplateTagIndexView(BaseAdminDocsView):
     template_name = 'admin_doc/template_tag_index.html'
 
     def get_context_data(self, **kwargs):
-        load_all_installed_template_libraries()
-
+        # Non-trivial TEMPLATES settings aren't supported (#24125).
+        engine = Engine.get_default()
         tags = []
-        app_libs = list(libraries.items())
-        builtin_libs = [(None, lib) for lib in builtins]
+        app_libs = sorted(list(engine.template_libraries.items()))
```

Comment:
I think list() is redundant with sorted(), e.g. https://github.com/django/django/pull/4666